### PR TITLE
Let actions be executed on gh hosted runners

### PR DESCRIPTION
**Pull request by FloThinksPi**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -42,15 +42,19 @@ jobs:
...
+        with:
+          go-version: 1.17.4
       - name: run golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
-          # Caching doesn't work on GH Enterprise w/ self-hosted runners
+          version: v1.43.0
+          skip-go-installation: true
           skip-pkg-cache: true
           skip-build-cache: true
```
Comment (johha) [inline .github/workflows/lint.yml:55]: @andy-paine suggested to disable caching. Can be also done in a separate PR.

**Review by johha — APPROVED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (FloThinksPi) [reply .github/workflows/lint.yml:55]: @johha  enable or disable ? currently it is disabled. Just the Comment for GHE is removed :) .
